src/templates/components/ManualInline.svelte

```diff
@@ -0,0 +1,105 @@
+<script lang="ts">
+	import { clickMacro } from '$lib/gam';
+	import '$templates/components/fonts/SansBold.css';
+	import '$templates/components/fonts/EgyptianBold.css';
+	import type { Tone } from '$lib/types/tones';
+	import ToneLogo from './ToneLogo.svelte';
+	import ArrowRight from './icons/ArrowRight.svelte';
+
+	export let baseUrl: string;
+	export let offerUrl: string;
+	export let tone: Tone;
+	export let image: string;
+	export let title: string;
+	export let linkText: string;
+</script>
+
+<header>
+	<a
+		href={clickMacro(baseUrl)}
+		class="logo"
+		data-link-name="title"
+		target="_top"
+	>
+		<ToneLogo {tone} />
+	</a>
+</header>
+<a
+	href={clickMacro(offerUrl)}
+	data-link-name="Offer 1 | {title}"
+	class="card"
+	target="_top"
+>
+	<img src={image} alt={title} />
+	<div class="text">
+		<h2><slot name="title" /></h2>
+		<p class="description"><slot name="text" /></p>
+		<div class="button">
```

Comment:
This button might be optional like with `manual-multiple`?

Reply:
Good point. The button now won't display if the text is blank